designs/capi-resources-custom-namespace.md

```diff
@@ -0,0 +1,63 @@
+# CAPI Resources in Custom Namespace
+
+## Introduction
+
+**Problem:** Customers managing large scale EKS-A clusters (hundreds or thousands of workload clusters for a given management cluster) expect to have difficulty managing all these resources if they are all in the same namespace. Specifically, they expect to encounter two problems:
+
+1. Difficulty troubleshooting issues for a single cluster when there are so many resources to sift through. 
+2. High levels of load on a bootstrap cluster when executing an upgrade, since the clusterctl move operation will take all the CAPI components inside a namespace and instatiate them on the bootstrap cluster. This can result in heavy and unnecessary load on the bootstrap cluster
```

Comment:
@maxdrib If I'm not mistaken, this problem is restricted to upgrades of the management cluster only?

Reply:
Yep! That's the only scenario. where we would be moving all the workload+management clusters' resources onto a bootstrap cluster

Reply:
How does namespaces help with this? Does CAPI already provide the ability to move only resources for a specific namespace?